### PR TITLE
Update of integration tests

### PR DESCRIPTION
**Pull request by lgrd**
# Update of integration tests
When we run `npm run itest`, most of the tests work (300/345) but we still need to fix some tests of the following classes :
- osrmResource
- pgrResource
- resource
- resourceManager
- smartPgrResource (will be removed from the project soon)
- valhallaResource
- service
- pgrSource
- smartroutingSource (will be removed from the project soon)
- sourceManager
- valhallaSource

### CODE REVIEW

File: test/functional/readme.md [hunk 1]
```diff
@@ -4,30 +4,24 @@ Cucumber sera utilisé afin de tester les APIs de Road2, et cela, dans leurs asp
...
-La feature `configurationTest.feature` permet de tester différentes configuration. Elle a l'avantage d'être accessible dès que l'image docker a été construite. La seconde `configurationComplementTest.feature` permet de tester plus de fonctionnalités. Pour que cette feature soit testable, il est nécessaire de construire une ressource PGR sur l'île-de-France. C'est pour cette raison que ces features sont séparées. 
+Les features `configuration/cucumber/features/conf*.feature` permettent de tester les fonctionnalités liées au chargement d'une configuration de Road2. 
 
-Afin de lancer les tests Cucumber, on suivra la procédure suivante:
+Afin de lancer ces tests, on suivra la procédure suivante:
 - lancer le serveur Road2 via docker-compose 
-- exécuter la commande `npm run ctest` via docker-compose
-- générer une ressource PGR sur l'île-de-France via docker-compose 
-- exécuter la commande `npm run ccftest` via docker-compose
+- exécuter la commande `npm run rtest` via docker-compose.
```
Comment (jmkerloch) [inline test/functional/readme.md:24]: It should be npm run ctest no ?
```suggestion
- exécuter la commande `npm run ctest` via docker-compose. 
```

**Review by jmkerloch — CHANGES_REQUESTED** [review]
@lgrd could you rebase to develop ? I have an issue with docker-compose build that seems to be corrected.

In general it would be better if all branches are rebased from develop before merging with PR.

**Comment by jmkerloch** [comment]
@lgrd I rebased locally and I have an error when launching itest. Do you have any idea of what could be wrong ? Maybe the test are not valid after a rebase.

```
> road2@2.0.0-DEVELOP itest
> nyc mocha --recursive './test/integration/mocha/**/*.js'
TypeError: Cannot read properties of undefined (reading 'schema')
    at new pgrSource (/home/docker/app/src/js/sources/pgrSource.js:21:69)
    at Suite.<anonymous> (/home/docker/app/test/integration/mocha/sources/integrationPgrSource.js:67:16)
```

**Comment by lgrd** [comment]
@jmkerloch some tests (from itest, like those of pgrSource class) are not ready. I think I will comment them until I fix them.

**Review by jmkerloch — APPROVED** [review]

**Pull request merged**